lib/presentation/widget/shared_space/shared_space_widget.dart

```diff
@@ -373,6 +374,7 @@ class _SharedSpaceWidgetState extends State<SharedSpaceWidget> {
 
   List<Widget> _contextMenuActionTiles(BuildContext context, SelectableElement<SharedSpaceNodeNested> sharedSpace) {
     return [
+      _renameWorkgroupAction(context, sharedSpace.element),
```

Comment:
no need to check the role to execute this action or not?

Reply:
oke i fixed it